README.md

```diff
@@ -11,3 +11,83 @@ TODO: separate make tasks to specify distro on build for local
 ```bash
 make dist
 ```
+
+## Usage Example
+
+### Provider
+
+For a test example you might not of configured your own certificates. If so
+please download the self-signed certs from `<your-domain>/ipa/config/ca.crt` and
+put save to `/etc/ipa/ca.crt`
+
+```tf
+provider "freeipa" {
+  host     = var.freeipa_host
+  username = var.freeipa_username
+  password = var.freeipa_password
+  base_dn  = var.freeipa_base_dn
+}
+```
+
+Arguments:
+* `host` - host name of IPA server, example *ipa.test.com* (string, required)
```

Comment:
Fairly pedantic here but might be better to use `example.com` or `example.net`, etc... as these are explicitly reserved for use in documentation. Where as `test.com` is not.